Xamarin.Forms.Xaml/MarkupExtensionParser.cs

```diff
@@ -24,6 +25,8 @@ public object Parse(string match, ref string remaining, IServiceProvider service
 				markupExtension = new OnIdiomExtension();
 			else if (match == "DataTemplate")
 				markupExtension = new DataTemplateExtension();
+			else if (match == "DependencyTemplate")
```

Comment:
Will remove this in order to revert the DependencyTemplateExtension implementation